catatonit.c

```diff
@@ -89,22 +91,22 @@ static void help(void)
 	usage();
 	fprintf(stderr, "\n");
 	fprintf(stderr, "options:\n");
-	fprintf(stderr, "  -h   Print this help page.\n");
-	fprintf(stderr, "  -L   Print license information.\n");
-	fprintf(stderr, "  -V   Print version information.\n");
+	fprintf(stderr, "  -h              Print this help page.\n");
+	fprintf(stderr, "  -L              Print license information.\n");
+	fprintf(stderr, "  -V, --version   Print version information.\n");
 	fprintf(stderr, "\n");
 	fprintf(stderr, "The source code can be found at <%s>.\n", PROGRAM_URL);
 	fprintf(stderr, "For bug reporting instructions, please see: <%s>.\n", PROGRAM_BUGURL);
 }
 
 static void version(void)
 {
-	fprintf(stderr, "%s version %s\n", PROGRAM_NAME, PROGRAM_VERSION);
+	fprintf(stdout, "tini version %s_%s\n", PROGRAM_VERSION, PROGRAM_NAME);
```

Comment:
tini version is intentional?

Reply:
Yes, that's intentional (unfortunately). Docker hard-codes the parsing of `docker-init --version` to tini. We either do this or we keep an out-of-tree patch for our docker packages, and we preferred this option as we consider it to be more portable.

Reply:
Technology is amazing! :smile_cat:

Reply:
Omg, thanks for the clarification